wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -12,10 +12,10 @@
         {% include "wagtailadmin/shared/side_panels.html" %}
     </div>
 
-    <button type="button" data-all-panels-toggle aria-expanded="true" aria-controls="">Collapse all sections</button>
-
     <form id="page-edit-form" action="{% url 'wagtailadmin_pages:add' content_type.app_label content_type.model parent_page.id %}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %} data-edit-form>
         {% csrf_token %}
+        <button type="button" data-all-panels-toggle aria-expanded="true" aria-controls="">Collapse all sections</button>
```

Comment:
suggestion:

```suggestion
        <button type="button" data-all-panels-toggle aria-expanded="true" aria-controls="">{% trans "Collapse all sections" %}</button>
```

we want all strings to be translateable

Reply:
ah thanks will sort that

Reply:
will need to make the js translatable too - or is that the data-attribute part that @lb- referred to?

Reply:
The text in JS should would also need translating. Will try to find references on how that is done

Reply:
Yep, as I understood it @lb-'s suggestion was to stash all of the necessary translatable text as `data-` attributes so that the Javascript code can refer to those for any text it needs to output, rather than have those strings embedded in the JS.

Reply:
hey - we do have JS translation utils but my reasoning was that we already have to put the 'default' label in the template - we may as well put the other state translations in there also. Instead of duplication of them in the JS.